### ISSUE TITLE
Running `okteto up --deploy` fails with nil pointer dereference in GithubActions pipeline

### ISSUE THREAD

**Issue by ivasilyev-servicetitan-com** [okteto/okteto#4170]
**Describe the bug**
`okteto up --deploy --command '/bin/sh' --command '-c' --command $TestCommand` fails with 
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x2274df1]

goroutine 1 [running]:
github.com/okteto/okteto/pkg/log/io.(*IOController).Out(...)
	github.com/okteto/okteto/pkg/log/io/io.go:49
github.com/okteto/okteto/pkg/cmd/build.(*OktetoBuilder).Run(0xc001f6e3a0, {0x354fa88, 0x510f840}, 0xc001a74f20, 0x0)
	github.com/okteto/okteto/pkg/cmd/build/build.go:91 +0x2f1
...
```
Running okteto version 2.25.1 on GithubActions pipeline.

**To Reproduce**
1. okteto context use $OktetoUri --token $OktetoPat --namespace $deploymentName
1. okteto deploy --remote --build --wait --var "GH_TOKEN=$env:GH_TOKEN"
1. okteto up --deploy --command '/bin/sh' --command '-c' --command $TestCommand

ERROR: 
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x2274df1]

goroutine 1 [running]:
github.com/okteto/okteto/pkg/log/io.(*IOController).Out(...)
	github.com/okteto/okteto/pkg/log/io/io.go:49
github.com/okteto/okteto/pkg/cmd/build.(*OktetoBuilder).Run(0xc001f6e3a0, {0x354fa88, 0x510f840}, 0xc001a74f20, 0x0)
	github.com/okteto/okteto/pkg/cmd/build/build.go:91 +0x2f1
github.com/okteto/okteto/cmd/build/remote.(*OktetoBuilder).Build(0xc001f539e0, {0x354fa88, 0x510f840}, 0xc001a74f20)
	github.com/okteto/okteto/cmd/build/remote/build.go:94 +0x1ce
github.com/okteto/okteto/cmd/deploy.(*remoteDeployCommand).deploy(0xc0002ea070, {0x354fa88, 0x510f840}, 0xc001a2e9c0)
	github.com/okteto/okteto/cmd/deploy/remote.go:242 +0x1393
github.com/okteto/okteto/cmd/deploy.(*DeployCommand).RunDeploy(0xc001c69ea0, {0x354fa88, 0x510f840}, 0xc001a2e9c0)
	github.com/okteto/okteto/cmd/deploy/deploy.go:383 +0xa15
github.com/okteto/okteto/cmd/up.(*upContext).deployApp(0xc001299500, {0x354fa88, 0x510f840}, 0xc0012a7208)
	github.com/okteto/okteto/cmd/up/up.go:602 +0x48c
github.com/okteto/okteto/cmd/up.Up.func1(0xc00185b800?, {0xc0008f6d20, 0x0, 0x7})
	github.com/okteto/okteto/cmd/up/up.go:293 +0x19c5
github.com/spf13/cobra.(*Command).execute(0xc00185b800, {0xc0008f6cb0, 0x7, 0x7})
	github.com/spf13/cobra@v1.7.0/command.go:940 +0x87c
github.com/spf13/cobra.(*Command).ExecuteC(0xc001295800)
	github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3a5
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.7.0/command.go:992
main.main()
	github.com/okteto/okteto/main.go:188 +0x10db
```

**Expected behavior**
No error.

**Desktop (please complete the following information):**
N/A

**Additional context**

**Comment by ifbyol** [comment]
Hi @ivasilyev-servicetitan-com thanks for reporting it, this was fixed as part of https://github.com/okteto/okteto/pull/4177 and today we have released the version `2.25.2` to fix it. Could you try it when you have a chance?

**Comment by ifbyol** [comment]
Closing this as it should be already fixed in `2.25.2`. If you still experience this issue, please, reopen it
